### ISSUE TITLE
User can assign Server to Client Steering DS from Delivery Services page

### ISSUE THREAD

**Issue by dpham692** [apache/trafficcontrol#4248]
## I'm submitting a ...

- [x] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [x] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:

In Traffic Portal, when you go to any Client Steering DS there isn't any option to add Server to the Delivery Service. 
<img width="1406" alt="Screen Shot 2019-12-31 at 11 46 10 AM" src="https://user-images.githubusercontent.com/21675397/71630986-6321d480-2bc3-11ea-9916-7993c3755489.png">
But user can go to the Delivery Services page and add Server from the outside. 
<img width="1400" alt="Screen Shot 2019-12-31 at 11 49 49 AM" src="https://user-images.githubusercontent.com/21675397/71631041-b431c880-2bc3-11ea-91d4-c98affc2cf9b.png">

## Expected / new behavior:

Should remove the Manage Servers for the Steering DS from Delivery Service page.

**Comment by mitchell852** [comment]
Good catch. Also `Manage Server Capabilities` should be removed from the context menu for Steering* delivery services.

**Comment by smalenfant** [comment]
This is also a problem when you go to "servers" and assigned delivery services. You can assign Client Steering services. When done, it creates some cryptic messages in atstscfg logs.

**Comment by rawlinp** [comment]
Ultimately, we should be preventing this from happening via the API.
